Fixup! Deprecation warning for sympy 1.7

cattabiani opened the pull request:
Fix #485

pramodk commented on nmodl/ode.py:
```diff
@@ -12,7 +12,10 @@
 # accessed through regular imports
-known_functions = import_module('sympy.printing.ccode').known_functions_C99
+if int(sp.__version__.replace('.','')) < 17:
+    known_functions = import_module('sympy.printing.ccode').known_functions_C99
```
versions like `1.6.1` like possible?

cattabiani replied:
yes, I noticed when I woke up this morning. Corrected

cattabiani commented:
please retest

pramodk reviewed (commented):
LGTM!

one comment though : our CI install version <1.6. I assume that is still necessary for tests? (to get same output?)

https://github.com/BlueBrain/nmodl/blob/d583ae80943b576f2498d1736dc7b0712c26fa0d/azure-pipelines.yml#L41

cattabiani commented:
> one comment though : our CI install version <1.6. I assume that is still necessary for tests? (to get same output?)

not at all! I fixed that with this: #441

I thought I removed that limit but, evidently, I forgot

cattabiani commented:
please retest

ohm314 reviewed (approved):
lgtm

alkino commented on nmodl/ode.py:
```diff
@@ -12,11 +12,15 @@
 # accessed through regular imports
-known_functions = import_module('sympy.printing.ccode').known_functions_C99
+major, minor = (int(v) for v in sp.__version__.split(".")[:2])
+if major >= 1 and minor >= 7:
```
This is not correct.

2.2 will not work here.

pramodk replied:
Stupid me! thanks Nico for pointing that!

We should convert this to integer and compare against it (`major*10+minor > 17`).

alkino replied:
Only if minor will never be > 9
